### ISSUE TITLE
parmgridgen.rb installation dependency issue

### ISSUE THREAD

**Issue by hadisfr** [mrklein/openfoam-os-x#45]
I'm getting some errors while trying to install by `brew`.
```console
$ brew install https://raw.githubusercontent.com/mrklein/openfoam-os-x/master/formulae/parmgridgen.rb
######################################################################## 100.0%
Error: parmgridgen: Unsupported special dependency :mpi
```
And after removing `  depends_on :mpi => :cc`:
```console
$ brew install parmgridgen.rb 
==> Downloading https://github.com/mrklein/ParMGridGen/archive/v0.0.2.tar.gz
Already downloaded: /Users/hadi/Library/Caches/Homebrew/downloads/fc1d32fa5314ed86ff952aa1fdc69104e84a148218347a2929bf1ee71031b444--ParMGridGen-0.0.2.tar.gz
==> make parallel
Last 15 lines from /Users/hadi/Library/Logs/Homebrew/parmgridgen/01.make:
a - setup.o
a - diffuse.o
a - grsetup.o
a - timer.o
a - coarsen.o
a - edge_refine.o
a - drivers.o
a - remap.o
a - iidxsort.o
a - ikeysort.o
a - var.o
make[3]: *** [../libIMparmetis.a] Error 1
make[2]: *** [default] Error 2
make[1]: *** [default] Error 2
make: *** [parallel] Error 2

Do not report this issue to Homebrew/brew or Homebrew/core!
```

Could you give me any hint please?

**Comment by mrklein** [comment]
Syntax was changed (quite a long time ago, it is funny, nobody reported the issue). I have pushed fix (279f553ddeffbfd904067b9a3533d8c078693c69).

**Comment by hadisfr** [comment]
Thank you! I'm gonna continue installation.
